dftimewolf/lib/module.py

```diff
@@ -117,6 +118,16 @@ def ModuleError(self, message: str, critical: bool=False) -> None:
       raise error
     self.logger.error(error.message)
 
+  def PublishMessage(self, message: str, is_error: bool = False) -> None:
+    """Sends a message to the CDM to disaply.
+
+    Args:
+      message: The message content.
+      is_error: True if the message is an error message, False otherwise."""
+    if self.state.cdm:
+      self.state.cdm.EnqueueMessage(self.name, message, is_error)
+    self.logger.info(message)
```

Comment:
Yeah that did cross my mind, but I opted away from it. There's no reason we couldn't have an optional logging level parameter, but I'm not sure it's necessary.

Reply:
I don't really know how that will look like in the end result so it's hard for me to tell. But the `success` statements should be kept, since they are the ones that indicate to the user where to find things (e.g. GRR flow IDs, timesketch URLs, etc.). If you're telling me that we only keep 5% of the lines we had before, then I think I can survive without any fancy coloring :)

Reply:
No logging is getting removed (it all still ends up in the debug log), but I think we could use any "success" log as an indicator that it should use PublishMessage instead.